### ISSUE TITLE
Support storing UnixFS 1.5 Owner

### ISSUE THREAD

**Issue by yuzhou-nj** [ipfs/kubo#10577]
### Checklist

- [X] My issue is specific & actionable.
- [X] I am not suggesting a protocol enhancement.
- [X] I have searched on the [issue tracker](https://github.com/ipfs/kubo/issues?q=is%3Aissue) for my issue.

### Description

The ipfs add command supports --preserve-mode and --preserve-mtime. Is it possible to also support --preserve-owner? This would allow changes made by chown to be preserved when adding files with ipfs add.

for example:
```
# ipfs add --preserve-mode --preserve-mtime test.json
added QmbwaAtVk282BkCwK18FDXniC26sTmZqSJvwRAVPAm3fV8 test.json

# chown oracle test.json
# ipfs add --preserve-mode --preserve-mtime test.json
added QmbwaAtVk282BkCwK18FDXniC26sTmZqSJvwRAVPAm3fV8 test.json  <--not changed
```

thank you!

**Comment by lidel** [comment]
No,  owner is not part of Unixfs 1.5 (only mode and mtime), so it is not possible atm. 

If you want to preserve owner information, wrap your data in a TAR, and store that.

Adding support for it would require IPIP spec proposal in https://github.com/ipfs/specs/ and requires https://github.com/ipfs/specs/pull/331 to land first.

I'm closing this, because there is no spec for Kubo to implement, but if you would like to kick-off discussion about  adding owner support to UnixFS specs, feel free to open issue in https://github.com/ipfs/specs/issues, or even propose an IPIP yourself.
